Cargo.toml

```diff
@@ -19,12 +19,15 @@ digest = "0.8"
 typenum = { version = "1.11", optional = true }
 arrayref = "0.3"
 subtle = { version = "2.2", default-features = false }
+serde =  { version = "1.0.104", features = ["derive"] }
```

Comment:
This should have `default-features = false` and then adding `std` features in `features` section.